daemon/authorization/get_album_song.py

```diff
@@ -0,0 +1,43 @@
+import urllib.request
+import json
+
+class Get_album_song:
+    
+    
+
+    def __init__(self,access_token):
+        """
+            Get Spotify catalog information for a single album.
+        """
+        self.headers = {
+            "Authorization": "Bearer " + access_token
+        } 
+# }
+    def get_album_song(self,album_id):
```

Comment:
I think this function is duplicated with getTracksByAlbumId in the class of SpotifyProvider.